Sources/ClientRuntime/Telemetry/Metrics/AsyncMeasurement.swift

```diff
@@ -16,7 +16,7 @@ public protocol AsyncMeasurementHandle {
     func stop()
 }
 
-public typealias LongAsyncMeasurement = AsyncMeasurement<Int64>
+public typealias LongAsyncMeasurement = AsyncMeasurement<Int>
```

Comment:
Per `AsyncMeasurement`, use the platform-independent `Int`: https://docs.swift.org/swift-book/documentation/the-swift-programming-language/thebasics/#Int